src/consent-manager.js

```diff
@@ -123,10 +128,11 @@ export default class ConsentManager {
     }
 
     saveConsents(){
-        if (this.consents === null)
+        let consents = this.consents
+        if (consents === null)
             deleteCookie(this.cookieName)
-        const v = JSON.stringify(this.consents)
-        setCookie(this.cookieName, v, this.config.cookieExpiresAfterDays || 120)
+        let cookieValue = consents !== null ? JSON.stringify(consents) : ""
```

Comment:
JSON.stringify on empty objects throws a syntax error in IE <= 10

Reply:
That's a very helpful statement to understand this line.
It should become an inline comment.